### ISSUE TITLE
Publish a newer version with latest cxf

### ISSUE THREAD

**Issue by rhtyd** [cloudsoft/winrm4j#84]
Dear authors, can you help do a release with latest cxf version. Your library fails for me, when using vmware sdk 6.5/6.7 and cxf 3.2.6. Please help - @neykov @bostko @aledsage

**Comment by rhtyd** [comment]
I've got so far as to apply following diff:

```
--- a/pom.xml
+++ b/pom.xml
@@ -53,7 +53,7 @@
 
   <properties>
     <!-- Compilation -->
-    <java.version>1.7</java.version>
+    <java.version>1.8</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
@@ -64,7 +64,7 @@
     <surefire.version>2.19.1</surefire.version>
     <testng.version>6.9.10</testng.version>
     <guava.version>18.0</guava.version>
-    <cxf.version>3.1.10</cxf.version>
+    <cxf.version>3.2.6</cxf.version>
     <httpcomponents.httpcore.version>4.4.4</httpcomponents.httpcore.version>

```
But I'm getting test failures that I'm unable to fix myself:
```
INFO: MockWebServer[39603] received request: POST /wsman HTTP/1.1 and responded: HTTP/1.1 200 OK
Aug 20, 2018 5:24:08 PM okhttp3.mockwebserver.MockWebServer$4 processOneRequest
INFO: MockWebServer[39603] received request: POST /wsman HTTP/1.1 and responded: HTTP/1.1 200 OK
Aug 20, 2018 5:24:08 PM okhttp3.mockwebserver.MockWebServer$4 processOneRequest
INFO: MockWebServer[39603] received request: POST /wsman HTTP/1.1 and responded: HTTP/1.1 200 OK
Aug 20, 2018 5:24:08 PM okhttp3.mockwebserver.MockWebServer$3 acceptConnections
INFO: MockWebServer[39603] done accepting connections: Socket closed
Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 1.751 sec <<< FAILURE! - in io.cloudsoft.winrm4j.client.WinRmClientRecordedTest
testRecording(io.cloudsoft.winrm4j.client.WinRmClientRecordedTest)  Time elapsed: 1.199 sec  <<< FAILURE!
javax.xml.ws.soap.SOAPFaultException: Response message did not contain proper response data. Expected: {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}DeleteResponse
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.doRequest(WinRmClientRecordedTest.java:85)
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.testRecording(WinRmClientRecordedTest.java:68)
Caused by: org.apache.cxf.endpoint.ClientImpl$IllegalEmptyResponseException: Response message did not contain proper response data. Expected: {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}DeleteResponse
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.doRequest(WinRmClientRecordedTest.java:85)
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.testRecording(WinRmClientRecordedTest.java:68)

testRecording(io.cloudsoft.winrm4j.client.WinRmClientRecordedTest)  Time elapsed: 0.338 sec  <<< FAILURE!
javax.xml.ws.soap.SOAPFaultException: Response message did not contain proper response data. Expected: {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}DeleteResponse
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.doRequest(WinRmClientRecordedTest.java:85)
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.testRecording(WinRmClientRecordedTest.java:68)
Caused by: org.apache.cxf.endpoint.ClientImpl$IllegalEmptyResponseException: Response message did not contain proper response data. Expected: {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}DeleteResponse
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.doRequest(WinRmClientRecordedTest.java:85)
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.testRecording(WinRmClientRecordedTest.java:68)


Results :

Failed tests: 
io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.testRecording(io.cloudsoft.winrm4j.client.WinRmClientRecordedTest)
  Run 1: WinRmClientRecordedTest.testRecording:68->doRequest:85 » SOAPFault Response me...
  Run 2: WinRmClientRecordedTest.testRecording:68->doRequest:85 » SOAPFault Response me...


Tests run: 1, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] WinRM Parent 0.6.0-SNAPSHOT ........................ SUCCESS [  0.341 s]
[INFO] WinRM Service ...................................... SUCCESS [  2.582 s]
[INFO] WinRM Client ....................................... FAILURE [  5.139 s]
[INFO] Java WinRM library 0.6.0-SNAPSHOT .................. SKIPPED

```

Hope this helps.

**Comment by aledsage** [comment]
@rhtyd we're working on this - main fix in https://github.com/cloudsoft/winrm4j/pull/89. We're aiming to release a new version with this fix soon.

**Comment by aledsage** [comment]
This is fixed in the 0.6.1 release, which uses CXF 3.2.7 (see https://github.com/cloudsoft/winrm4j/pull/89).
